### DISCUSSION TITLE
mkcert: command not found - Fedora 34

### DISCUSSION THREAD

**Discussion by kennylajara** [FiloSottile/mkcert#445] [Q&A] [answered]
I use Fedora 34 and installed the dependencies, then compiled the it by my self by following the instructions on the README.md file and I get the message `mkcert: command not found`

**Comment by igorskv** [comment, marked as answer]
```
wget -O mkcert https://github.com/FiloSottile/mkcert/releases/download/v1.4.3/mkcert-v1.4.3-linux-amd64
chmod +x  mkcert
sudo mv mkcert /usr/local/bin
```

**Comment by HishamAbonjim** [comment]
i have a similar issue when using sudo mkcert , 

but when calling mkcert it does work,
